app/controllers/error.js

```diff
@@ -1,6 +1,6 @@
 errorResponse = (req, res, err, statusCode, message) => {
 	const userError = { message: message };
-	const systemError = err ? err?.message ?? err : message;
+	const systemError = err ? err?.name + " " + err?.message ?? err : message;
```

Comment:
Does the Nullish coalescing operator`??` still work when there's a space `" "` in the middle?

Reply:
You are right, it does not. I have broken up the logic